src/extensions/lens-extension.ts

```diff
@@ -14,35 +13,17 @@ export interface LensExtensionManifest {
   renderer?: string; // path to %ext/dist/renderer.js
 }
 
-export interface LensExtensionStoreModel {
-  isEnabled: boolean;
-}
-
-export class LensExtension<S extends ExtensionStore<LensExtensionStoreModel> = any> {
-  protected store: S;
+export class LensExtension {
   readonly manifest: LensExtensionManifest;
   readonly manifestPath: string;
   readonly isBundled: boolean;
 
+  @observable isEnabled = false;
```

Comment:
Should this be `private`? Does it need to be observable?

Reply:
Yes, it needs to be observable (used in `extension.whenEnabled`)

Reply:
Added `private` modifier.